mobile/src/components/common/category/category.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+
+import { categoryKeyToImage } from '~/common/maps/maps';
+import { CategoryImage, Text, View } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type Props = {
+  keyName: string;
+  name: string;
+  color?: string;
+  isActive?: boolean;
```

Comment:
Looks like we still need is Active here, then let's not reuse it in CourseCategory component, make it separate
```suggestion
  color: string;
```